pkg/kev/converter/kubernetes/project_service_test.go

```diff
@@ -1254,712 +1240,659 @@ var _ = Describe("ProjectService", func() {
-
-		When("not defined by label", func() {
-			It("returns default value", func() {
-				expected, _ := durationStrToSecondsInt(config.DefaultProbeTimeout)
-				Expect(projectService.readinessProbeTimeout()).To(Equal(*expected))
-			})
-		})
-	})
-
-	Describe("readinessProbeInitialDelay", func() {
-		When("defined via labels", func() {
-			delay := "30s"
-
-			BeforeEach(func() {
-				labels = composego.Labels{
-					config.LabelWorkloadReadinessProbeType:         config.ProbeTypeExec.String(),
-					config.LabelWorkloadReadinessProbeInitialDelay: delay,
-				}
-			})
-
-			It("returns label value", func() {
-				Expect(projectService.readinessProbeInitialDelay()).To(BeEquivalentTo(30))
-			})
-		})
-
-		When("not defined by label", func() {
-			It("returns default value", func() {
-				expected, _ := durationStrToSecondsInt(config.DefaultProbeInitialDelay)
-				Expect(projectService.readinessProbeInitialDelay()).To(Equal(*expected))
-			})
-		})
-	})
-
-	Describe("readinessProbeRetries", func() {
-		When("defined via labels", func() {
-			retries := "3"
-
-			BeforeEach(func() {
-				labels = composego.Labels{
-					config.LabelWorkloadReadinessProbeType:    config.ProbeTypeExec.String(),
-					config.LabelWorkloadReadinessProbeRetries: retries,
-				}
-			})
-
-			It("returns label value", func() {
-				expected := 3
-				Expect(projectService.readinessProbeRetries()).To(BeEquivalentTo(expected))
-			})
-		})
-
-		When("not defined by label", func() {
-			It("returns default value", func() {
-				Expect(projectService.readinessProbeRetries()).To(BeEquivalentTo(config.DefaultProbeRetries))
-			})
-		})
 	})
 
-	Describe("readinessProbeDisabled", func() {
-		When("defined via labels with valid (truth) value", func() {
-			BeforeEach(func() {
-				labels = composego.Labels{
-					config.LabelWorkloadReadinessProbeType: config.ProbeTypeNone.String(),
-				}
-			})
-
-			It("returns label value", func() {
-				pt, err := projectService.readinessProbeType()
-				Expect(err).To(Succeed())
-				Expect(*pt).To(Equal(config.ProbeTypeNone))
-			})
-
-			It("returns a nil probe", func() {
-				probe, err := projectService.readinessProbe()
-				Expect(err).To(Succeed())
-				Expect(probe).To(BeNil())
-			})
-		})
-
-		When("defined via labels with invalid (non truthy) value", func() {
-			disabled := "FOO"
-
-			BeforeEach(func() {
-				labels = composego.Labels{
-					config.LabelWorkloadReadinessProbeType: disabled,
-				}
-			})
-
-			It("returns an error", func() {
-				p, err := projectService.readinessProbe()
-				Expect(err).To(HaveOccurred())
-				Expect(err.Error()).To(ContainSubstring("not a supported readiness probe type"))
-				Expect(p).To(BeNil())
-			})
-		})
-
-		When("not defined via labels at all", func() {
-			It("returns default value - disable by default", func() {
-				p, err := projectService.readinessProbe()
-				Expect(err).To(Succeed())
-				Expect(p).To(BeNil())
-			})
-		})
-	})
+	// 	Describe("livenessProbeCommand", func() {
```

Comment:
Are these commented out specs something we want to keep?

Reply:
yes, we still want to implement them later again but right now there is no point changing them just to change it back later IMO